bundles/binding/org.openhab.binding.frontiersiliconradio/src/main/java/org/openhab/binding/frontiersiliconradio/internal/FrontierSiliconRadio.java

```diff
@@ -186,6 +188,37 @@ public void setMode(int mode) {
 			logger.error("request failed: " + REQUEST_SET_MODE + " = " + params);
 		}
 	}
+	
+		/**
+	 * Read the radios operating eq preset
+	 * 
+	 * @return operating eq preset. 
+	 */
+	public int getEq() {
+		try {
+			FrontierSiliconRadioApiResult result = conn.doRequest(REQUEST_GET_EQ);
+			return result.getValueU32AsInt();
+		} catch (Exception e) {
+			logger.error("request failed: " + REQUEST_GET_EQ);
+		}
+		return 0;
+	}
+
+	/**
+	 * Set the radio operating eq preset
+	 * 
+	 * @param eq
+	 *           0,1,2,3,4,5,6
+	 */
+	public void setEq(int eq) {
+		final String params = "value=" + eq;
+		try {
+			conn.doRequest(REQUEST_SET_EQ, params);
+			return;
+		} catch (Exception e) {
+			logger.error("request failed: " + REQUEST_SET_EQ + " = " + params);
```

Comment:
decrease log level to WARN or INFO since this error won't harm the whole platform